diff --git a/lib/cf-socket.c b/lib/cf-socket.c
index 273258fea..77af54b5a 100644
--- a/lib/cf-socket.c
+++ b/lib/cf-socket.c
@@ -1187,7 +1187,7 @@ static CURLcode cf_socket_open(struct Curl_cfilter *cf,
   }
 #endif
 
-#ifndef CURL_USE_SOCK_NONBLOCK
+#if 0 //ndef CURL_USE_SOCK_NONBLOCK
   /* Set socket non-blocking, must be a non-blocking socket for
    * a non-blocking connect. */
   error = curlx_nonblock(ctx->sock, TRUE);
